Manual.md

```diff
@@ -0,0 +1,145 @@
+
+```ruby
+require 'package' # must occour within each `.rb`
+			      # Notice the newline
+class Template < Package # Notice the capitals, EG: 'I3' - would be used for an 'i3' package
+  description 'The template script' # Notice the indent, should contain no more than one line of text
+  homepage '#' # Notice the same indent, EG: 'https://i3wm.org/' - Would be used for an 'i3' Package
+  version 'version#-revision#' # EG: 4.18.2-1 - Where '4.18.2' is the version and '1' is the script revision
+  compatibility '#' # Can contain 'all', or a list of supported architectures each seperated by a space
+  source_url '#' # The URL from which the source of the package can be downloaded
+  source_sha256 '#' # The `sha256` checksum of the downloaded source pcakage
+                  # Notice the newline
+  depends_on '#' # Soft where this package depends on
+  depends_on '#' => :build # Only required when the package is built from source
+  				  # Notice the newline
+  self.prebuild # For sed operations, not required
+    system '#' # Replace '#' with a sed command
+  end
+  self.patch # For patch operations, not required
+    system '#' # Replace '#' with a patch command
+  end
+				  # Notice the newline
+  self.build # Contains the commands which compile the package, required if package con be compiled
+    system '#' # Replace '#' with commands to compile the package 
+    system '#' # Should contain something like, "./configure --prefix=/usr/local"
+    system '#' # Use incase additions commands are required
+    system '#' # Should contain something like, "'make -j$(nproc)'"
+  end
+				  # Notice the newline
+  self.preinstall # For pre-install conditions, not required
+    system '#'
+  end
+				  # Notice the newline
+  self.install # For make install, required if package can be compiled
+    system '#' # Should contain something like, "'make', "DESTDIR=#{CREW_DEST_DIR}", 'install'"
+  end
+				  # Notice the newline
+  self.postinstall # For post-install messages or operations, not required
+    system '#' 
+  end
+end
+```
+***
+# Footnotes
+
+<a name="Arch">**Arch**</a>: Chromebrew packages for `x86_64`, `i686`, `armv7l`, and `aarch64`, try and make your package work on all of these architectures.  
+
+<a name="More">**More**</a>: Must come from an official source, not a distro's source archive. Should contain the version number<sup>(Which must be equal to the version variable)</sup>.  
+
+NOTE: All rules can have exceptions, if ***REQUIRED***, exceptions to the rules should be avoided at all costs.
+
+<a name="=">`CREW_OPTIONS`</a>: Equal to `--prefix=/usr/local --libdir=/usr/local/lib --mandir=/usr/local/share/man --build=armv7l-cros-linux-gnueabihf --host=armv7l-cros-linux-gnueabihf --target=#{CREW_BUILD}`
+
+This Manual is heavily based off <a hrel="https://github.com/void-linux/void-packages/blob/master/Manual.md"><code>void-packages/Manual.md</code></a>
+
+`#`'s are used as placeholders/wild-cards within this Manual.
```

Comment:
This might actually be good to put at the top to avoid confusion as it's referenced throughout the manual.